package-lock.json

```diff
@@ -8790,4 +8790,4 @@
       "dev": true
     }
   }
-}
+}
```

Comment:
Please don't include these in the PR. Ideally, configure VSCode to ensure newlines are added to the end of files when you save them!